AFQ/viz/utils.py

```diff
@@ -472,7 +472,6 @@ def bbox(img):
 def trim(im):
     bg = Image.new(im.mode, im.size, im.getpixel((0, 0)))
     diff = ImageChops.difference(im, bg)
-    diff = ImageChops.add(diff, diff, 2.0, -100)
```

Comment:
What's up with this change?

Reply:
this is an unrelated tweak to the code for making plots. this method removes whitespace around the edges of an image, and I think this line of code is not necessary at all. Let me remove the next line of code too, actually